### PR TITLE
Create Agile-SIG.md

### PR DESCRIPTION
**Pull request by nielsdrost**

### CODE REVIEW

File: proposals/2020-fall/Agile-SIG.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+## Which two persons will act as SIG Lead?
+- Berend Weel
+- Niels Drost
+
+## What is the SIG’s mission?
+
+The mission of the Agile SIG is to introduce, teach and train in Agile methodologies.
+We want Agile to become common knowledge in the center and aim for engineers/MT/DT/Operations to use it when appropriate.
+This will contribute to the team work within the center and to the restructuring and scaling phase we will enter in the near future.
+In addition the SIG acs as a platform for the exhange of experiences between the different teams at the center.
```
Comment (bpmweel) [inline proposals/2020-fall/Agile-SIG.md]: acts

File: proposals/2020-fall/Agile-SIG.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+Agile-SIG@esciencecenter.nl (Agile SIG)
+
+## SIG outcomes in the period April 2020 - October 2020 (for existing SIGs)
+
+The Agile SIG has become the discussion platform for how to succesfully apply Agile methods to working at the eScience Center.
+Due to the Corona crisis, involvement of external people is postponed.
+
+We are also working on setting up an Introduction to Agile course. This is progessing nicely.
+
```
Comment (bpmweel) [inline proposals/2020-fall/Agile-SIG.md:31]: Should we add the link? https://github.com/escience-academy/lesson-agile-escience

File: proposals/2020-fall/Agile-SIG.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+The mission of the Agile SIG is to introduce, teach and train in Agile methodologies.
+We want Agile to become common knowledge in the center and aim for engineers/MT/DT/Operations to use it when appropriate.
+This will contribute to the team work within the center and to the restructuring and scaling phase we will enter in the near future.
+In addition the SIG acs as a platform for the exhange of experiences between the different teams at the center.
+
+## What is the GitHub repository of the SIG use?
+
+https://github.com/nlesc-sigs/Agile-SIG
+
```
Comment (bpmweel) [inline proposals/2020-fall/Agile-SIG.md:20]: This repo does not actually exist, do we use it? Should we make it?

File: proposals/2020-fall/Agile-SIG.md (hunk 1 above)
Comment (bpmweel) [reply proposals/2020-fall/Agile-SIG.md]: I'm also thinking maybe this should be part of the main mission of the agile sig, perhaps:

The mission of the Agile SIG is to introduce, discuss and teach Agile methodologies within the eScience Center

**Review by bpmweel — COMMENTED** [review]
I added a few comments, on the whole it looks very good.

File: proposals/2020-fall/Agile-SIG.md (hunk 3 above)
Comment (nielsdrost) [reply proposals/2020-fall/Agile-SIG.md:20]: yes :-)

Comment (nielsdrost) [reply proposals/2020-fall/Agile-SIG.md:20]: no we do not use it, do you think we should?

File: proposals/2020-fall/Agile-SIG.md (hunk 1 above)
Comment (nielsdrost) [reply proposals/2020-fall/Agile-SIG.md]: yes, done

File: proposals/2020-fall/Agile-SIG.md (hunk 2 above)
Comment (nielsdrost) [reply proposals/2020-fall/Agile-SIG.md:31]: yes, done

**Pull request merged**